File: ModelAutomi/transizionepda.cpp
#include "transizionepda.h"

TransizionePDA::TransizionePDA(StatoPDA *s, char i, char h, const std::string &nh):Transizione(s, i), head(h)
{
    if(nh.length() <= 2)
        newHead = nh;
}
char TransizionePDA::getHead(){
  return head;
}
std::string TransizionePDA::getNewHead(){
  return newHead;
}

File: progettoP2/src/progettoP2/TransizionePDA.java
public class TransizionePDA extends Transizione{
	private char head;
	private String newHead;
	
	public TransizionePDA(Stato st, char i, char h, String nh){
		super(st, i);
		head = h;
		
		if(nh.length() <= 2){
			newHead = nh;
		}
	}
	
	char getHead(){
		return head;
	}
	
	String getNewHead(){
		return newHead;
	}
}
